bindings/wasm/examples/editor.js

```diff
@@ -299,7 +301,14 @@ require(['vs/editor/editor.main'], async function() {
       addEdit(button);
     }
   }
-  switchTo(currentName);
+  if (window.location.hash.length > 0) {
+    const name = unescape(window.location.hash.substring(1));
+    if (exampleFunctions.get(name) != null || getScript(name) != null)
+      switchTo(name);
```

Comment:
Is this `if` necessary?

Reply:
yeah it is not really needed, will remove